demo/config/cluster2/spire/spire-controller-manager-config.yaml

```diff
@@ -2,8 +2,8 @@ apiVersion: spire.spiffe.io/v1alpha1
 kind: ControllerManagerConfig
 metrics:
   bindAddress: 127.0.0.1:8082
-healthProbe:
-  bindAddress: 127.0.0.1:8083
+health:
+  healthProbeBindAddress: 127.0.0.1:8083
```

Comment:
Hmm, this feels like a breaking change....

Reply:
that is already broken, if you try to load health using previews library version that will not be loaded you can see that using old library, because the expected heal https://github.com/kubernetes-sigs/controller-runtime/blob/v0.14.6/pkg/config/v1alpha1/types.go#L67-L69
I found this when testing new code.